Controllers.test/CustomerController_UnitTesting.cs

```diff
@@ -0,0 +1,103 @@
+using AutoFixture;
+using AutoMapper;
+using FluentAssertions;
+using FoodOrderApi.Controllers;
+using FoodOrderApi.DataProvider;
+using FoodOrderApi.Mappings;
+using FoodOrderApi.Model.Domain;
+using FoodOrderApi.Model.DTO;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.Extensions.Logging;
+using Moq;
+
+namespace Controllers.test
+{
+    public class CustomerController_UnitTesting
```

Comment:
Class names generally should not contains `_`.

Reply:
Updated and checked the class names.